src/connector/with_options_source.yaml

```diff
@@ -798,45 +453,16 @@ PulsarOauthCommon:
     field_type: String
     required: true
   - name: oauth.scope
-    field_type: Option < String >
+    field_type: String
```

Comment:
Previously, `PulsarOauthCommon` is not inlined, because its `field_type` is Option.